style.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+::after,
+::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.navbar {
+  position: relative;
+  top: 10;
+  left: 10px;
+  width: 100%;
+  height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.menu {
+  display: none;
+}
+
+.nave-toggle {
+  margin-right: 20px;
+}
+
+.page-1 {
+  position: relative;
+  background-color: #fff;
+  border-bottom-left-radius: 100px 70px;
+  overflow-x: hidden;
+}
+
+.p1-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  padding-top: 50px;
+}
+
+.p1-description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.p1-connect {
+  color: #7f8cff;
+  font-weight: 500px;
+  font-style: normal;
+  font-size: 16px;
+  text-decoration: none;
+  line-height: 24px;
+  text-transform: uppercase;
+  letter-spacing: 1;
+}
+
+.p1-link {
+  display: flex;
+  width: clamp(12rem, 12vw, 15rem);
+  justify-content: space-between;
+  margin-bottom: 1rem;
+  list-style-type: none;
+}
```

Comment:
- [ ] Also I suggest that you align the social links correctly as required by the previous reviewer. This will make your site look more intentional

Your Design | Figma Design
-- | --
![image](https://user-images.githubusercontent.com/29071984/176952017-b063cf8c-88e7-447a-bf10-40400510cf9d.png) | ![image](https://user-images.githubusercontent.com/29071984/176952102-bee6127d-e909-4123-9c74-a262c94680e1.png)

[OPTIONAL]
- [ ] While you are at that, I suggest you take a look at the bottom left border radius of this section as shown in the Figma design and do the same with yours.